DigitalLearningSolutions.Data.Tests/Services/FaqsServiceTests.cs

```diff
@@ -154,5 +155,38 @@ public void GetPublishedFaqsForTargetGroup_returns_empty_list_when_data_service_
                     .MustHaveHappenedOnceExactly();
             }
         }
+
+        [Test]
+        public void GetAllFaqs_calls_data_service_method_and_returns_expected_results()
+        {
+            //Given
+            var expectedFaqs = Builder<Faq>.CreateListOfSize(10)

```

Comment:
Given that this service method is a wrapper service only we would not expect the data from the data service to be altered in any way.
With that it would be worth checking that these `expectedFaqs` equal `results`.